flowey/flowey_lib_hvlite/src/build_openhcl_igvm_from_recipe.rs

```diff
@@ -54,11 +54,20 @@ pub enum IgvmManifestPath {
     LocalOnlyCustom(PathBuf),
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub enum InitrdRootfsPath {
```

Comment:
nit: here and elsewhere, I would suggest using the slightly more verbose name `InitrdRootfsConfigPath`, to make it marginally more clear we're talking about the `rootfs.config` files here.

it took me a sec to realize this wasn't related to the pre-packaged initrd layers themselves